sync/src/lib.rs

```diff
@@ -50,7 +50,7 @@
 //! 	);
 //! 	let client = Client::new(
 //!			ClientConfig::default(),
-//!			ethereum::new_frontier(),
+//!			&ethereum::new_frontier(),
```

Comment:
is this directly related to snapshotting? if not, perhaps a list of any other refactoring/changes made would be helpful, plus comments to guide through the PR would speed along the review process